Core/OxidEeEvents.php

```diff
@@ -386,6 +257,32 @@ private static function _addPaymentMethod($oPayment)
         );
     }
 
+    /**
+     * Adds the payment method meta data to the database.
+     *
+     * @param object $oPaymentXml
+     *
+     * @since 1.2.0
+     */
+    private static function _addPaymentMethodMetaData($oPaymentXml)
+    {
+        $sPaymentId = (string) $oPaymentXml->oxid;
```

Comment:
If i understand you create an xml config and then store it as key value?

Reply:
Correct.